packages/alfa-rules/src/sia-r77/rule.ts

```diff
@@ -0,0 +1,86 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Element, Namespace } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Table, Cell } from "@siteimprove/alfa-table";
+import { Criterion, Technique } from "@siteimprove/alfa-wcag";
+import { Page } from "@siteimprove/alfa-web";
+
+import { expectation } from "../common/expectation";
+
+import { hasRole, isIgnored, isPerceivable } from "../common/predicate";
+
+const { isElement, hasName, hasNamespace } = Element;
+const { and, not } = Predicate;
+
+export default Rule.Atomic.of<Page, Element>({
+  uri: "https://alfa.siteimprove.com/rules/sia-r77",
+  requirements: [Criterion.of("1.3.1"), Technique.of("H43")],
+  evaluate({ device, document }) {
+    const data = new Map<Element, [cell: Cell, table: Table]>();
```

Comment:
```suggestion
    const data = Map.empty<Element, Cell>();
```
Since the table is not re-used in the expectation, no need to store it.